apps/artwork/components/related_artists/artists.jade

```diff
@@ -0,0 +1,21 @@
+each artist in relatedArtists
```

Comment:
Not necessarily something that needs addressing now, but this should probably be made into a component now that this template is in two places.